05_syntax_lab/06.py

```diff
@@ -1,9 +1,15 @@
-"""
-Write a program that randomizes 2 numbers
-and calculates their least common multiplier,
-that is the smallest number that is divisable
-by both.
-For example if the numbers were 4 and 6,
-program should print 12
-"""
-
+################### Part 6
+r1 = randint(1, 10)
+r2 = randint(1, 10)
+maximum = r1 * r2
+print('r1 = %d r2 = %d max quotient = %d' % (r1, r2, maximum))
+i = 1
+min_quotient = []
+while i <= maximum:
+    if (i % r1 == 0) and (i % r2 == 0):
+        min_quotient.append(i)
+        #print('{}'.format(min_quotient))
+    i += 1
+
+print('%d' % min(min_quotient))
+

```

Comment:
This algorithm will be quite slow if the numbers are big (not just 10*9). Could you find and implement a faster one?